CHANGELOG.md

```diff
@@ -1,3 +1,211 @@
+# 5.0.0
+
+This version includes big changes around upgrading to selenium-webdriver 3.0.x.
+See the [selenium-webdriver changelog](https://github.com/SeleniumHQ/selenium/blob/master/javascript/node/selenium-webdriver/CHANGES.md). Some of the breaking change highlights are summarized below:
+
+- Minimum node version 6.9.x for selenium-webdriver dependency.
+- Jasmine typescript users will need to include the `@types/jasmine`. This
+  is to not cause conflicts with for mocha typings.
+- Removed `protractor.wrapDriver()`.
+- You can no longer use `repl` command from within `browser.pause()`. Instead,
+  use `browser.explore()` to directly enter the `repl`.
+- Methods removed: `WebDriver.prototype.isElementPresent`,
+  `WebElement.prototype.getRawId`, `WebElement.prototype.getInnerHtml`, and
+  `WebElement.prototype.getOuterHtml`.
+- Selenium-webdriver `ErrorCodes` have been deprecated.
+- Adding cookies have been changed from
+  `browser.manage().addCookie('testcookie', 'Jane-1234');` to
+  `browser.manage().addCookie({name:'testcookie', value: 'Jane-1234'});`
+- When testing with Firefox 47, use `marionette: false` to use legacy Firefox
+  driver.
+
+For the 5.0.0 release, we are still using the selenium standalone server 2.53.1
+and recommend using Firefox 47.
+
+## Features
+
+- ([ec93c4a](https://github.com/angular/protractor/commit/ec93c4ab882991410ad9d3f52d87c0f5ec947641))
+  chore(cli): **breaking change** throw errors on unknown flags (#3921)
+
+  closes #3216
+- ([bc58332](https://github.com/angular/protractor/commit/bc583321a233453fc2b89472013b2ec3e1d6b6f9))
+  feat(rootEl): ***breaking change*** auto-detect the root element better (#3928)
+
+  This is a breaking change because it changes the default root element behavior
+ and removes the `config.useAllAngular2AppRoots` flag.  Modern angular apps now
+ default to using all app hooks, and ng1 apps now check several places, notably
+ the element the app bootstraps to.
+
+  closes #1742
+- ([604fdbf](https://github.com/angular/protractor/commit/604fdbf064cc2785a2e745747beeaeb45d27f8ed))
+  cleanup(config): **breaking change** Remove redundant sauceProxy config (#3868)
```

Comment:
It removes the `sauceProxy` config field, and uses `webDriverProxy` when creating the SauceLabs client.